src/export/InstanceExporter.ts

```diff
@@ -717,7 +717,14 @@ export class InstanceExporter implements Fishable {
     if (
       this.shouldSetMetaProfile(instanceDef) &&
       isResource &&
-      instanceOfStructureDefinition.derivation === 'constraint'
+      instanceOfStructureDefinition.derivation === 'constraint' &&
+      instanceOfStructureDefinition.elements.some(
+        el =>
+          el.path === `${instanceOfStructureDefinition.pathType}.meta` &&
+          el.type?.[0].code === 'Meta' &&
+          el.max === '1' &&
+          (el.base?.max == null || el.base.max === '1')
+      )
```

Comment:
Yeah, good idea. For some reason I thought we used that somewhere else too. But you're right. Fixed in f644004.

Reply:
Note - I also removed the check for `isResource` because the purpose of that was to ensure that whatever we set it on had a `meta`, but now we check that explicitly.